src/sentry/api/endpoints/project_details.py

```diff
@@ -252,9 +252,19 @@ def put(self, request, project):
             project.update_option('mail:subject_template', result['subjectTemplate'])
 
         if result.get('isSubscribed'):
-            UserOption.objects.set_value(request.user, project, 'mail:alert', 1)
+            UserOption.objects.set_value(
```

Comment:
these calls changed syntax because I modified the order of the args in set_value to fit organization